model/roast.go

```diff
@@ -153,6 +153,9 @@ func GetRoast(id int) (roast *RoastResult, err error) {
 		msg := RoastError{}
 		err = errorRows.Scan(&msg.Hash, &msg.Row, &msg.Column, &msg.Engine, &msg.Name, &msg.Description)
 		roast.Errors = append(roast.Errors, msg)
+		if err != nil {
+			return
+		}
```

Comment:
This should be checked _before_ the `roast.Errors` are appended.

Also, add this error check after row `175` too.